src/arrayMethodJoin.js

```diff
@@ -4,8 +4,23 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let joinedString = '';
+    const divide = separator;
+
+    for (let i = 0; i < this.length; i++) {
+      const element = this[i];
+
+      if (element !== undefined && element !== null) {
+        joinedString += `${element}`;
```

Comment:
There's no need to use `template string`

```suggestion
        joinedString += element;
```